Backpack/src/main/java/net/skyscanner/backpack/util/BpkViewPumpContextWrapper.kt

```diff
@@ -0,0 +1,15 @@
+package net.skyscanner.backpack.util
+
+import android.content.Context
+import android.content.ContextWrapper
+import io.github.inflationx.viewpump.ViewPumpContextWrapper
+
+object BpkViewPumpContextWrapper {
+    fun wrap(context: Context): ContextWrapper {
+      return if (android.os.Build.VERSION.SDK_INT > android.os.Build.VERSION_CODES.P) {
+        ContextWrapper(context)
```

Comment:
You can change the return to be `Context`  and just return `context` here instead of wrapping. A `ContextWrapper` is also a context.